client/my-sites/post-type-list/post-actions-ellipsis-menu/edit.jsx

```diff
@@ -21,8 +22,12 @@ function PostActionsEllipsisMenuEdit( { translate, siteId, canEdit, status, edit
 		return null;
 	}
 
+	function bumpStat() {
```

Comment:
>Curious, any reason you didn't convert this to extend `Component` as well?

It probably should, at least so far as we want to avoid creating functions for every render call ([`react/jsx-no-bind` describes this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md)). I think I was just trying to avoid excessive refactoring.